Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -268,7 +268,7 @@ open class MediaAttachment: NSTextAttachment {
         ensureImageIsUpToDate(in: textContainer)
 
         if image == nil {
-            return .zero
+            return CGRect(x: 0, y: 0, width: 20, height: 20)
```

Comment:
Is this to make sure that something is show visually even when an image is not set?

Reply:
If yes, can we create a constant called minimum size, or something similar?

Reply:
Sounds good!